site/src/components/image.js

```diff
@@ -23,23 +23,20 @@ const getOverlapProps = mode => ({
 
 function Image({ zoom, isZoomed = false, width = '75vw', ...props }) {
   const [mode] = useColorMode()
-  const Wrapper = props.fluid || props.fixed ? GatsbyImage : 'img'
   if (zoom) {
     return (
       <Zoom {...getOverlapProps(mode)}>
```

Comment:
This was a nice little "happy accident." It _just worked_ without needing to keep my previous hackery.